examples/pom.xml

```diff
@@ -260,7 +260,7 @@
         <dependency>
             <groupId>jakarta.ws.rs</groupId>
             <artifactId>jakarta.ws.rs-api</artifactId>
-            <version>3.0.0-SNAPSHOT</version>
+            <version>3.0.0-M1</version>
```

Comment:
Shouldn't this be `3.0.0`?

Reply:
Had concerns about this failing somewhere since we are not using -Pstaging for examples. I could try to change this, but all the other deps here will not be final.